Gaijinizer patch/Map131.txt

```diff
@@ -103,11 +107,11 @@ It was about rotten so far.#MTLed
 > CONTEXT: Map131/events/5/pages/0/44/Dialogue/enemy10
 > CONTEXT: Map131/events/8/pages/0/44/Dialogue/enemy10
 > CONTEXT: Map131/events/9/pages/0/44/Dialogue/enemy10
-......Well, I think I felt that I was recent.
-Even though the fighting ability was enough, I
-could not give up my companions
-In the end, a man who has been trapped by the
-familiars .........#MTLed
+...Aah, that reminds me, there were some
+interesting ones recently. Their fighting
+strength was more than sufficient to escape, but
+they couldn't abandon their companions and ended
+up as food for my familiars.#Ron
```

Comment:
Minor break for flow:
>_**...Aah, that reminds me, there were some
interesting ones recently. Their fighting
strength was more than sufficient to escape...**_
>
>_**But they couldn't abandon their companions, and
ended up as food for my familiars.**_